src/app/zap-templates/zcl/data-model/chip/device-energy-management-cluster.xml

```diff
@@ -113,74 +127,104 @@ limitations under the License.
     <item name="PoolPump" value="0x0D"/>
     <item name="Other" value="0xFF"/>
   </enum>
+
   <enum name="ESAStateEnum" type="enum8" apiMaturity="provisional">
     <cluster code="0x0098"/>
     <item name="Offline" value="0x00"/>
     <item name="Online" value="0x01"/>
     <item name="Fault" value="0x02"/>
-    <item name="UserOptOut" value="0x03"/>
-    <item name="PowerAdjustActive" value="0x04"/>
-    <item name="Paused" value="0x05"/>
+    <item name="PowerAdjustActive" value="0x03"/>
+    <item name="Paused" value="0x04"/>
   </enum>
+
   <enum name="CauseEnum" type="enum8" apiMaturity="provisional">
     <cluster code="0x0098"/>
     <item name="NormalCompletion" value="0x00"/>
     <item name="Offline" value="0x01"/>
     <item name="Fault" value="0x02"/>
     <item name="UserOptOut" value="0x03"/>
+    <item name="Cancelled" value="0x04"/>
+  </enum>
+
+  <enum name="AdjustmentCauseEnum" type="enum8">
+    <cluster code="0x0098"/>
+    <item name="LocalOptimization" value="0x00"/>
+    <item name="GridOptimization" value="0x01"/>
+  </enum>
+
+  <enum name="ForecastUpdateReasonEnum" type="enum8">
+    <cluster code="0x0098"/>
+    <item name="InternalOptimization" value="0x00"/>
+    <item name="LocalOptimization" value="0x01"/>
+    <item name="GridOptimization" value="0x02"/>
```

Comment:
Again, in the spec these have spaces.  Please fix the spec.

Reply:
https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/8766 raised